### ISSUE TITLE
Hirschsprung's disease OMIM xrefs

### ISSUE THREAD

**Issue by sbello** [DiseaseOntology/HumanDiseaseOntology#291]
The OMIM xrefs for Hirschsprung's Disease (DOID:10487) are all for OMIM susceptibility terms. These should be split out into contributes_to_condition relationships. The IDs are:

OMIM:606874	{Hirschsprung disease, susceptibility to, 6}
OMIM:611644	{Hirschsprung disease, susceptibility to, 9}
OMIM:613711	{Hirschsprung disease, susceptibility to, 3}
OMIM:600156	{Hirschsprung disease, susceptibility to, 5}
OMIM:142623	{Hirschsprung disease, susceptibility to, 1}
OMIM:600155	{Hirschsprung disease, susceptibility to, 2}
OMIM:608462	{Hirschsprung disease, susceptibility to, 8}
OMIM:606875	{Hirschsprung disease, susceptibility to, 7}
OMIM:613712	{Hirschsprung disease, susceptibility to, 4}

**Comment by lschriml** [comment]
Look at for review of OMIM IDs for susceptibility

**Comment by sbello** [comment]
see ticket #711

**Comment by melodyswen** [comment]
@sbello question: are you going to add the rest of the hirschprung susceptibility to terms? There are 9 of them now. Thanks!

**Comment by sbello** [comment]
Yes, I will get to them as soon as I can

**Comment by sbello** [comment]
Circling back to this as part of the UniProt list

Notes from ticket #711 

For the susceptibility terms, in general, we can treat the different kinds as 'defined phenotype sets' and create xrefs to the DO term. Thus we would not encode the types of susceptibility entries.

For Hirschsprung disease
--> based on Genetics Home Reference,
https://ghr.nlm.nih.gov/condition/hirschsprung-disease
-->. we can create two subtypes
short-segment Hirschsprung disease
long-segment Hirschsprung disease

There are two main types of Hirschsprung disease, known as short-segment disease and long-segment disease, which are defined by the region of the intestine lacking nerve cells. In short-segment disease, nerve cells are missing from only the last segment of the large intestine. This type is most common, occurring in approximately 80 percent of people with Hirschsprung disease.

Cheers,
Lynn

The 2008 reference (https://www.ncbi.nlm.nih.gov/pubmed/17965226) mentions the long and short form classification but also mentions what I think is a second classification scheme described as "Four HSCR variants have been reported: (1) total colonic aganglionosis (TCA, 3–8% of cases)17; (2) total intestinal HSCR when the whole bowel is involved17; (3) ultra-short segment HSCR involving the distal rectum below the pelvic floor and the anus18; (4) suspended HSCR, a controversial condition, where a portion of the colon is aganglionic above a normal distal segment.".
However, I don't see much followup for the second scheme in other papers. The long/short split is very commonly mentioned
See https://www.ncbi.nlm.nih.gov/pubmed/22174542 for a more recent update on the genetics of HSCR
There is also an interesting 2011 paper on gene-environment interactions in HSCR https://www.ncbi.nlm.nih.gov/pubmed/26997034

**Comment by sbello** [comment]
Currently most of the OMIM IDs are still in as part of the susceptibility load. 
The phenotypic separation into long form and short form HSCR appears to depend on the mix of genetic factors involved so we can't match different OMIM terms to the 2 phenotypic types.

Was the plan to leave the OMIM IDs as susceptibility cross references and NOT create DO terms for these. Looking at the notes I think that was what we decided but want to confirm before I proceed.
